corehq/celery_monitoring/signals.py

```diff
@@ -0,0 +1,53 @@
+import datetime
```

Comment:
If you want to be my hero, add imports for both ```absolute_import``` and ```unicode_literals``` here.  Otherwise I have to add them during my weekly ```modernize``` run.

Reply:
🙇